scripts/ansible/roles/linux/docker/tasks/ci-setup.yml

```diff
@@ -38,10 +38,12 @@
     name: linux/intel
     tasks_from: sgx-packages.yml
 
+# az-dcap-client is not avaialble for Ubuntu 22.04
```

Comment:
```suggestion
# az-dcap-client is not available for Ubuntu 22.04
```

Reply:
It has been requested, and the changes are relatively minor: https://github.com/microsoft/Azure-DCAP-Client/pull/182

Reply:
That's awesome. Thanks @achamayou !